cachet-tools/status-updater-template.yaml

```diff
@@ -0,0 +1,167 @@
+      including a build configuration, application deployment configuration and secure
+      communication using edge TLS.
+    template.openshift.io/support-url: https://access.redhat.com
+    version: 1.0.0
+  creationTimestamp: "2019-09-27T16:48:54Z"
+  labels:
+    samples.operator.openshift.io/managed: "true"
+  name: status-updater
+  # namespace: openshift
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: Status updater for updating Cachet Status page
+      labels: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+  spec:
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      app: ${APPLICATION_NAME} # not in eap template
+      deploymentConfig: ${APPLICATION_NAME}
+    sessionAffinity: None
+    type: ClusetIP # not in eap template
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+    namespace: ${APPLICATION_NAME}
+  spec:
+    failedBuildsHistoryLimit: 5
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${APPLICATION_NAME}:latest
+    source:
+      git:
+        ref: update-cachet
+        uri: git@github.com:thearifismail/black-box-tester.git
+      type: Git
+    strategy:
+      dockerStrategy:
+        from:
+          kind: ImageStreamTag
+          name: python-36-centos7:latest
+      type: Docker
+      # type: Source
+    triggers:
+    - github:
+        secret: vG5wco-HyDjKdbhsGERy
+      type: GitHub
+    - generic:
+        secret: TxWDWvspJ3boffU4-61x
+      type: Generic
+    - imageChange:
+        type: ImageChange
+    - type: ConfigChange
+
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+  spec:
+    replicas: 1
+    selector:
+      deploymentconfig: ${APPLICATION_NAME}
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: ${APPLICATION_NAME}
+          deploymentconfig: ${APPLICATION_NAME}
+        name: ${APPLICATION_NAME}
+      spec:
+        containers:
+        - env:
+          - name: CACHET_HOSTNAME
+            value: ${CACHET_HOSTNAME}
+          - name: CACHET_TOKEN
+            value: ${CACHET_TOKEN}
+          - name: INSIGHTS_SERVER
+            value: ${INSIGHTS_SERVER}
+          - name: INSIGHTS_USERNAME
+            value: ${INSIGHTS_USERNAME}
+          - name: INSIGHTS_PASSWORD
+            value: ${INSIGHTS_PASSWORD}
+          - name: PYTHONWARNINGS
+            value: ${}
+          # image: image-registry.openshift-image-registry.svc:5000/${APPLICATION_NAME}/${APPLICATION_NAME}@sha256:5cad79e8647107c7b1769246f611240e6aa3e78ad0db1648ad4490d3c5d23c45
+          image: ${APPLICATION_NAME}
```

Comment:
do you need a tag here?

Reply:
Oops.  Shouldn't have been checked in.  Removed the file, still working on it.  This review was for the update_status script to get in.